api/fetch-pokemon-data.jsx

```diff
@@ -0,0 +1,14 @@
+import { fetchData } from '../utils/fetch-data';
+
+export const fetchPokemonData = async ({ name = '' } = {}) => {
+  try {
+    const data = await fetchData({
+      URL: `https://pokeapi.co/api/v2/pokemon/${name}`,
+      query: {},
```

Comment:
No need to pass this

Reply:
It's not getting the results if I remove the query parameter